### DISCUSSION TITLE
Is it possible to use Supabase with Unity?

### DISCUSSION THREAD

**Discussion by beratcmn** [supabase/supabase#6494] [Questions] [answered]
If so, how I can use? 
I am trying to develop a Multiplayer game and Player Auth is an unknown area for me. I don't know even know where to begin. All I can think is creating a REST server with something like FastAPI and sending requests to that server. But that approach doesn't feel right so I'm asking you. Is it possible to use Supabase in my Unity Game?
Thanks.

**Comment by steve-chavez** [comment]
Unity uses C# right? This could help:

- https://github.com/supabase-community/supabase-csharp

**Reply by beratcmn** [reply]
Yes Unity uses C#. Thank you, I'm checking it right now.

**Reply by ravenzachary** [reply]
> Yes Unity uses C#. Thank you, I'm checking it right now.

@beratcmn: Any luck with Unity? We'd like to do the same.

**Reply by feremabraz** [reply]
Here's a full tutorial for Supabase with Unity: https://www.youtube.com/watch?v=S0hTwtsUWcM
It's very well done, uses Nuget to fetch the latest build and does the Auth stuff.

**Comment by kamyker** [comment, marked as answer]
https://github.com/kamyker/supabase-unity
